sw/airborne/firmwares/rotorcraft/oneloop/oneloop_andi.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2023 Tomaso De Ponti <tmldeponti@tudelft.nl>
+ *
+ * This file is part of paparazzi
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+/** @file "firmwares/rotorcraft/oneloop/oneloop_andi.h"
+ * @author Tomaso De Ponti <tmldeponti@tudelft.nl>
+ * One loop (Guidance + Stabilization) ANDI controller for the rotating wing drone RW3C
+ */
+
+#ifndef ONELOOP_ANDI_H
+#define ONELOOP_ANDI_H
+
+#include "firmwares/rotorcraft/stabilization.h"
+#include "firmwares/rotorcraft/stabilization/stabilization_attitude_common_int.h"
+#include "firmwares/rotorcraft/stabilization/stabilization_attitude_ref_quat_int.h"
+
+#define ANDI_G_SCALING 1000.0f
+
+extern float att_ref[3];
```

Comment:
These are a lot of variables in the global namespace. Please use an andi struct instead.

Reply:
you are right! I tried to clean up the variables and I defined new structs. Let me know if now it is fine.